content/200-concepts/100-components/02-prisma-client/140-debugging.mdx

```diff
@@ -17,18 +17,15 @@ You can enable debugging output in Prisma Client via the [`DEBUG`](../../../refe
 Here are examples for setting these debugging options in bash:
 
 ```terminal
-# enable only `prisma:engine`-level debugging output
-export DEBUG="engine"
-
 # enable only `prisma:client`-level debugging output
 export DEBUG="prisma:client"
 
 # enable both `prisma-client`- and `engine`-level debugging output
 export DEBUG="prisma:client,prisma:engine"
 ```
 
-If you want to enable all debugging options, you can also set `DEBUG` to `*`:
+If you want to enable all debugging options, you can also set `DEBUG` to `prisma*`:
 
 ```terminal
-export DEBUG="*"
+export DEBUG="prisma*"
```

Comment:
I do not think replacing this is a good choice, it should probably just be listed above as the most wide catching Prisma level option. We have been telling people to use DEBUG=* for ages in all our issues and will keep doing so as it gives the full picture (also including stuff that might influence Prisma from the outside (like loading env vars in dotenv etc).

Reply:
```suggestion
export DEBUG="*"
```